plugins/CoreHome/Controller.php

```diff
@@ -111,7 +111,7 @@ public function showInContext()
         $controllerName = Common::getRequestVar('moduleToLoad');
         $actionName     = Common::getRequestVar('actionToLoad', 'index');
 
-        if($controllerName == 'API') {
+        if($controllerName == 'API' || ($controllerName == 'Proxy' && $actionName == 'callApi')) {
             throw new Exception("Showing API requests in context is not supported for security reasons. Please change query parameter 'moduleToLoad'.");
```

Comment:
I know that it's not your code, but would be great to create an own exception for it instead of this long description eg. ```NotAllowedRequestException``` or ```NotAllowedtApiCallException```